components/feed/FeedPage.tsx

```diff
@@ -1,27 +1,89 @@
+"use client";
+
+import { useState } from "react";
+
+import { postFeedContent } from "../../pages/api/feedContent/[id]";
+
 import FeedItem from "./FeedItem";
 
+import type { User } from "next-auth";
+
+export type FeedComment = {
+  id: string;
+  username: string;
+  comment: string;
+  createTime: Date;
+};
+
 export type FeedItemContent = {
   id: string;
-  data: { username: string; content: string };
+  data: {
+    username: string;
+    content: string;
+    comments: FeedComment[];
+    createTime: Date;
+  };
 };
 
 const FeedPage = ({
   data,
   showLinks,
+  user,
 }: {
   data: FeedItemContent[];
   showLinks?: boolean;
-}) => (
-  <ul>
-    {data.map((feedItem) => (
-      <FeedItem
-        key={feedItem.id}
-        username={feedItem.data.username}
-        feedContent={feedItem.data.content}
-        showLink={showLinks}
-      />
-    ))}
-  </ul>
-);
+  user: User;
+}) => {
+  const [postText, setPostText] = useState("");
+
+  async function handleSubmit() {
+    await postFeedContent(user.id, `${postText}`);
+    window.location.reload();
+  }
+
+  return (
+    <div>
+      <form>
+        <div className="w-full mb-4 border border-gray-200 rounded-lg bg-gray-50 dark:bg-gray-700 dark:border-gray-600">
+          <div className="px-4 py-2 bg-white rounded-t-lg dark:bg-gray-800">
+            <textarea
+              id="comment"
+              className="w-full px-0 text-sm text-gray-900 bg-white border-0 dark:bg-gray-800 focus:ring-0 dark:text-white dark:placeholder-gray-400"
+              placeholder="Write your musical thoughts..."
+              value={postText}
+              onChange={(e) => setPostText(e.target.value)}
+              required
+            />
+          </div>
+          <div className="flex items-center justify-between px-3 py-2 border-t dark:border-gray-600">
+            <button
+              // eslint-disable-next-line @typescript-eslint/no-misused-promises
+              onClick={() => handleSubmit()}
```

Comment:
```suggestion
              onClick={() => void handleSubmit()}
```

This should fix the esLint rule so you dont have to disable it